bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -91,17 +63,17 @@ The thing has the following configuration parameters:
 
 All devices support some of the following channels:
 
-| group     | channel          |type                    | description                  | things supporting this channel                 |
-|-----------|----------------- |------------------------|------------------------------|------------------------------------------------|
-| actuator  | output           | Switch                 | Power device on or off       | P100, P105, P110, P115, L510, L530, L900, L920 |
-|           | brightness       | Dimmer                 | Brightness 0-100%            | L510, L530, L900                               |
-|           | colorTemperature | Number                 | White-Color-Temp 2500-6500K  | L510, L530, L900                               |
-|           | color            | Color                  | Color                        | L530, L900                                     |
-| device    | wifiSignal       | system.signal-strength | WiFi-quality-level           | P100, P105, P110, P115, L510, L530, L900, L920 |
-|           | onTime           | Number:Time            | seconds output is on         | P100, P105, P110, P115, L510, L530, L900, L920 |
-| energy    | actualPower      | Number:Power           | actual Power (Watt)          | P110, P115                                     |
-|           | todayEnergyUsage | Number:Energy          | used energy today (Wh)       | P110, P115                                     |
-|           | todayRuntime     | Number:Time            | seconds output was on today  | P110, P115                                     |
+| group     | channel          |type                    | description                  | things supporting this channel                                   |
+|-----------|----------------- |------------------------|------------------------------|------------------------------------------------------------------|
+| actuator  | output           | Switch                 | Power device on or off       | P100, P105, P110, P115, L510, L530, L610, L630, L900, L920, L930 |
+|           | brightness       | Dimmer                 | Brightness 0-100%            | L510, L530, L610, L630, L900                                     |
+|           | colorTemperature | Number                 | White-Color-Temp 2500-6500K  | L510, L530, L610, L630, L900                                     |
+|           | color            | Color                  | Color                        | L530, L630, L900                                                 |
+| device    | wifiSignal       | system.signal-strength | WiFi-quality-level           | P100, P105, P110, P115, L510, L530, L610, L630, L900, L920, L930 |
```

Comment:
`system.signal-strength` is a channel type while this column is mentioning the item type expected when linking the channel. To be checked what is the item type for `system.signal-strength` but probably `Number`.